WalletWasabi/Helpers/IoHelpers.cs

```diff
@@ -272,6 +272,22 @@ public static void OpenFileInTextEditor(string filePath)
 				}
 				else
 				{
+					if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+					{
+						if (!EnvironmentHelpers.IsFileTypeAssociated("json"))
+						{
+							// Open file using Notepad.
+							using (Process process = Process.Start(new ProcessStartInfo
+							{
+								FileName = "notepad.exe",
+								Arguments = filePath,
+								CreateNoWindow = true,
+								UseShellExecute = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
Here we know the platform is Windows because we checked it before. 
```suggestion
								UseShellExecute = false
```